src/main/java/io/camunda/testing/assertions/MessageAssert.java

```diff
@@ -99,12 +77,34 @@ public MessageAssert hasNotBeenCorrelated() {
     return this;
   }
 
+  /**
+   * Verifies the expectation that a message start event has been correlated
+   *
+   * @return this {@link MessageAssert}
+   */
+  public MessageAssert hasCreatedProcessInstance() {
+    final boolean isCorrelated =
+        StreamFilter.messageStartEventSubscription(recordStreamSource)
+            .withMessageKey(actual.getMessageKey())
+            .withRejectionType(RejectionType.NULL_VAL)
+            .withIntent(MessageStartEventSubscriptionIntent.CORRELATED)
+            .stream()
+            .findFirst()
+            .isPresent();
+
+    assertThat(isCorrelated)
+        .withFailMessage("Message with key %d was not correlated", actual.getMessageKey())
```

Comment:
:x: The error message is incorrect or imprecise. The message may have been correlated, but not to a message start event.
Suggestions:
* Message with key %d was not correlated to a message start event
* Message with key %d did not lead to the creation of a process instance.